presto-common/src/main/java/com/facebook/presto/common/function/OperatorType.java

```diff
@@ -95,4 +95,46 @@ public boolean isArithmeticOperator()
     {
         return this.equals(ADD) || this.equals(SUBTRACT) || this.equals(MULTIPLY) || this.equals(DIVIDE) || this.equals(MODULUS);
     }
+
+    public OperatorType flip()
```

Comment:
Put this as the first commit and use these functions directly rather than do a later refactor.

Reply:
yep, get it updated